### PR TITLE
Add Zuul configuration

### LINKED ISSUE #1124: Migrate to Zuul CI
# Description
Travis-CI is no longer a good option for Anitya, because of the limitation and now when travis-ci.org instance is no longer working we need to find a new CI. Because some other projects are using Zuul and I'm familiar with it, I decided to move over to it.

## Requirements
* Create a Zuul configuration file
* Add Anitya to Zuul CI
* Add checks for Zuul to mergify
* Update documentation
* (Optional) Solve the codecov issue

## How this will make Anitya better
The Anitya will again have a CI running.

### PR DESCRIPTION
**Pull request by Zlopez**
Closes #1124 

Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

File: .zuul.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#- job:
```
Comment (morucci) [inline .zuul.yml:1]: let's rename the file to .zuul.yaml

Comment (Zlopez) [reply .zuul.yml]: It looks like this helped

File: .zuul.yaml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+- project:
+    check:
+      jobs:
+        - tox-lint
+        - tox-format
+        - tox-python37
+        - tox-python38
+# Anitya wasn't tested with python 3.9 yet and there are few failing tests now
+# TODO: Fix tests for python 3.9
+#        - tox-python39
```
Comment (AdamSaleh) [inline .zuul.yaml:157]: I think I have some fixes in my stale pr for mypy setup, so once this is merged I can look into that.

**Review by AdamSaleh — APPROVED** [review]
Looks good. Like the use of pods, should be faster than full vm's.

**Comment by Zlopez** [comment]
I want to setup mergify and update docs before merging this

File: .zuul.yaml (hunk 2 above)
Comment (Zlopez) [reply .zuul.yaml:157]: Thanks, this will be nice

I was a little nervous without working CI

**Comment by Zlopez** [comment]
@softwarefactory-project-zuul help

**Pull request merged**